README.md

```diff
@@ -2192,3 +2195,7 @@ I'm a front-end newbie and a cat person.
 
 My name is [Dan](https://github.com/4-13).
 I'm wanna be frontend developer.
+<<<<<<< HEAD
+>>>>>>> master
+=======
+>>>>>>> master
```

Comment:
Seems like you forgot remove lines which was automatically added.

Reply:
that is, I should delete them and send the PR again?

Reply:
@Pozzitive11  Probably yes, or do you add them by yourself ?


One more thing. Mark conversation resolved only  in case when you are sure that all discussions were finished

Reply:
I seem to have fixed everything, please check